weather@mockturtl/files/weather@mockturtl/po/nl.po

```diff
@@ -345,1008 +371,1143 @@ msgstr ""
+#: 3.0/utils.js:204
 msgid "Thursday"
 msgstr "Donderdag"
 
-#: 3.0/utils.js:204 3.8/utils.js:161
+#: 3.0/utils.js:204
 msgid "Friday"
 msgstr "Vrijdag"
 
-#: 3.0/utils.js:204 3.8/utils.js:161
+#: 3.0/utils.js:204
 msgid "Saturday"
 msgstr "Zaterdag"
 
-#: 3.0/utils.js:301 3.8/utils.js:254
+#: 3.0/utils.js:301 3.8/utils.js:142
 msgid "Calm"
 msgstr "Rustig"
 
-#: 3.0/utils.js:304 3.8/utils.js:257
+#: 3.0/utils.js:304 3.8/utils.js:145
 msgid "Light air"
 msgstr "Zeer licht briesje"
 
-#: 3.0/utils.js:307 3.8/utils.js:260
+#: 3.0/utils.js:307 3.8/utils.js:148
 msgid "Light breeze"
 msgstr "Licht briesje"
 
-#: 3.0/utils.js:310 3.8/utils.js:263
+#: 3.0/utils.js:310 3.8/utils.js:151
 msgid "Gentle breeze"
 msgstr "Zacht briesje"
 
-#: 3.0/utils.js:313 3.8/utils.js:266
+#: 3.0/utils.js:313 3.8/utils.js:154
 msgid "Moderate breeze"
 msgstr "Gematigd briesje"
 
-#: 3.0/utils.js:316 3.8/utils.js:269
+#: 3.0/utils.js:316 3.8/utils.js:157
 msgid "Fresh breeze"
 msgstr "Fris briesje"
 
-#: 3.0/utils.js:319 3.8/utils.js:272
+#: 3.0/utils.js:319 3.8/utils.js:160
 msgid "Strong breeze"
 msgstr "Sterk briesje"
 
-#: 3.0/utils.js:322 3.8/utils.js:275
+#: 3.0/utils.js:322 3.8/utils.js:163
 msgid "Near gale"
 msgstr "Bijna storm"
 
-#: 3.0/utils.js:325 3.8/utils.js:278
+#: 3.0/utils.js:325 3.8/utils.js:166
 msgid "Gale"
 msgstr "Storm"
 
-#: 3.0/utils.js:328 3.8/utils.js:281
+#: 3.0/utils.js:328 3.8/utils.js:169
 msgid "Strong gale"
 msgstr "Harde storm"
 
-#: 3.0/utils.js:334 3.8/utils.js:287
+#: 3.0/utils.js:334 3.8/utils.js:175
 msgid "Violent storm"
 msgstr "Heftige storm"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "N"
 msgstr "N"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "NE"
 msgstr "NO"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "E"
 msgstr "O"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "SE"
 msgstr "ZO"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "S"
 msgstr "Z"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "SW"
 msgstr "ZW"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "W"
 msgstr "W"
 
-#: 3.0/utils.js:429 3.8/utils.js:382
+#: 3.0/utils.js:429
 msgid "NW"
 msgstr "NW"
 
+#: 3.8/weatherbit.js:230
+msgid "API key is doesn't provide access to Hourly Weather, skipping"
+msgstr "API sleutel geeft geen toegang tot het uurlijks weer,overslaan"
+
+#: 3.8/main.js:103
+msgid "This provider requires an API key to operate"
+msgstr "Deze provider heeft een API sleutel nodig om te kunnen werken"
+
+#: 3.8/main.js:136 3.8/uiBar.js:36
+msgid "As of {lastUpdatedTime}"
+msgstr "Vanaf {laatst bijgewerkt tijdstip}"
+
+#: 3.8/uiBar.js:42
+msgid "{distance}{distanceUnit} from you"
+msgstr "afstand}{afstandseenheid} van jou"
+
+#: 3.8/yahoo.js:109 3.8/yahoo.js:116
+msgid "Please install '{missingPackage}', then refresh manually."
+msgstr "Installeer a.u.b '{ontbrekend pakket}', ververs daarna handmatig."
```

Comment:
Please do not change anything inside {} brackets because they will be string formatted out based on that name.

Reply:
Going to modify it and submit a new pull request